RingbufferAsyncAddWithBackoffStressTest.whenShortTTLAndBigBuffer

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-1 completed error expected null, but was:<com.hazelcast.ringbuffer.StaleSequenceException: sequence:5814 is too small. The current headSequence is:5818>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotNull(Assert.java:755)
    at org.junit.Assert.assertNull(Assert.java:737)
    at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:60)
    at com.hazelcast.ringbuffer.impl.RingbufferAsyncAddWithBackoffStressTest.test(RingbufferAsyncAddWithBackoffStressTest.java:81)
    at com.hazelcast.ringbuffer.impl.RingbufferAsyncAddWithBackoffStressTest.whenShortTTLAndBigBuffer(RingbufferAsyncAddWithBackoffStressTest.java:57)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast/709/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAsyncAddWithBackoffStressTest/whenShortTTLAndBigBuffer/

--- a/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAsyncAddWithBackoffStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAsyncAddWithBackoffStressTest.java
@@ -37,7 +37,7 @@ public class RingbufferAsyncAddWithBackoffStressTest extends HazelcastTestSuppor
     public void whenTTLEnabled() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(200 * 1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(2);
         test(ringbufferConfig);
     }
 
@@ -45,7 +45,7 @@ public class RingbufferAsyncAddWithBackoffStressTest extends HazelcastTestSuppor
     public void whenLongTTLAndSmallBuffer() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(30);
         test(ringbufferConfig);
     }
 
@@ -53,7 +53,7 @@ public class RingbufferAsyncAddWithBackoffStressTest extends HazelcastTestSuppor
     public void whenShortTTLAndBigBuffer() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(20 * 1000 * 1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(2);
         test(ringbufferConfig);
     }
 

